stdlib/sqlite3/dbapi2.pyi

```diff
@@ -312,27 +313,27 @@ else:
     def register_adapter(type: type[_T], caster: _Adapter[_T], /) -> None: ...
     def register_converter(name: str, converter: _Converter, /) -> None: ...
 
-class _AggregateProtocol(Protocol):
-    def step(self, value: int, /) -> object: ...
-    def finalize(self) -> int: ...
+class _AggregateProtocol(Protocol[_SQLType]):
```

Comment:
I guess this doesn't really need to be generic, but for consistency and since it shouldn't make a difference. Happy to switch it over though.